src/plugins/intel_gpu/src/kernel_selector/core/cl_kernels/experimental_detectron_detection_output_ref.cl

```diff
@@ -133,21 +133,28 @@ KERNEL(eddo_ref_stage_0)
     size_t roi_idx = get_global_id(0);
     size_t class_idx = get_global_id(1);
 
-    INPUT_TYPE4 box = vload4(roi_idx, boxes);
+    INPUT_TYPE4 box;
+    box[0] = boxes[INPUT0_GET_INDEX(roi_idx, 0, 0, 0)];
+    box[1] = boxes[INPUT0_GET_INDEX(roi_idx, 1, 0, 0)];
+    box[2] = boxes[INPUT0_GET_INDEX(roi_idx, 2, 0, 0)];
+    box[3] = boxes[INPUT0_GET_INDEX(roi_idx, 3, 0, 0)];
 
     if (any(islessequal(box.hi - box.lo, ZERO2))) {
         const int refined_offset = roi_count * class_idx + roi_idx;
         refined_scores[refined_offset] = ZERO;
     } else {
-        const int offset = NUM_CLASSES * roi_idx + class_idx;
-
         // width & height of box
         INPUT_TYPE2 box_size = (box.hi - box.lo + COORDINATE_OFFSET);
 
         // center location of box
         const INPUT_TYPE2 center = box.lo + HALF_ONE * box_size;
 
-        const INPUT_TYPE4 delta = vload4(offset, deltas) / DELTA_WEIGHTS;
+        INPUT_TYPE4 delta;
+        delta[0] = deltas[INPUT1_GET_INDEX(roi_idx, class_idx * 4, 0, 0)];
+        delta[1] = deltas[INPUT1_GET_INDEX(roi_idx, class_idx * 4 + 1, 0, 0)];
+        delta[2] = deltas[INPUT1_GET_INDEX(roi_idx, class_idx * 4 + 2, 0, 0)];
+        delta[3] = deltas[INPUT1_GET_INDEX(roi_idx, class_idx * 4 + 3, 0, 0)];
```

Comment:
shouldn't here be like
INPUT1_GET_INDEX(roi_idx, class_idx , 1, 0)
INPUT1_GET_INDEX(roi_idx, class_idx , 2, 0)
INPUT1_GET_INDEX(roi_idx, class_idx , 3 ,0)